fetch.js

```diff
@@ -106,6 +106,9 @@
 
     if (blobSupport) {
       this.blob = function() {
+        if (!this._bodyBlob) {
+          throw new Error('XMLHttpRequest missing blob support')
+        }
```

Comment:
I don't think this is correct. A `Response` can be constructed with a String and accessed from the blob mixing.